radio/src/targets/common/arm/stm32/stm32_spi.cpp

```diff
@@ -290,7 +316,7 @@ uint8_t stm32_spi_transfer_byte(const stm32_spi_t* spi, uint8_t out)
 }
 
 uint16_t stm32_spi_transfer_bytes(const stm32_spi_t* spi, const uint8_t* out,
-				  uint8_t* in, uint16_t length)
+				  uint8_t* in, size_t length)
```

Comment:
This seems like a unnecessary change. Besides, formatting with tabs is actually prohibited by clangd configuration.